src/index.ts

```diff
@@ -28,7 +29,8 @@ try {
             core.warning(`Codecov warning: ${err.message}`);
           }
         })
-        .then(() => {
+        .then((status) => {
+          console.log(status);
           unlinkFile();
```

Comment:
What’s the purpose of creating the script only to remove it? What were the original thoughts behind it?

Reply:
The original intent was that the script did not live inside the repository, but it was getting curled in. This would then write the output to file and run. Thanks for bringing this up, that code has been removed now.